packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -72,6 +73,7 @@ export default class DedicatedServerDashboard {
     });
 
     this.loadStatistics();
+    this.advices = this.loadAdvices();
```

Comment:
Hmm `loadAdvices` return Promise call in a specfic case, no use of `then` block ?

Reply:
no not here. loadAdvices is passed to a component. the component takes promise and waits for it to resolve. the component can take the promise that resolves to array or array directly